html/browser.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <title>DTDI Mineral Observation Map</title>
 
-    <script src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.1/handlebars.min.js"></script>
+    <script src="handlebars.min.js"></script>
```

Comment:
1) Why remove the handlebars CDN reference?
2) Did you add the handlebar.min.js to the repo?